src/spaces/prophoto-linear.js

```diff
@@ -19,6 +19,7 @@ const fromXYZ_M = [
 
 export default new RGBColorSpace({
 	id: "prophoto-linear",
+	cssId: "--prophoto-rgb-linear",
```

Comment:
Should this be `--prophoto-linear`? I went with `--prophoto-rgb-linear` for consistency with `color(prophoto-rgb)`.